Example/InstanceID/Tests/FIRInstanceIDBackupExcludedPlistTest.m

```diff
@@ -113,18 +114,25 @@ - (void)testMovePlistToApplicationSupportDirectoryFailure {
 
   NSDictionary *newPlistContents = @{@"world" : @"hello"};
   [self.plist writeDictionary:newPlistContents error:nil];
+
   XCTAssertEqualObjects(newPlistContents, [self.plist contentAsDictionary]);
 
   // The new file should still be written to the Documents folder.
   XCTAssertFalse([self isPlistInApplicationSupportDirectory]);
   XCTAssertTrue([self isPlistInDocumentsDirectory]);
+#endif
 }
 
 #pragma mark - Private Helpers
 
 - (BOOL)isPlistInApplicationSupportDirectory {
+#if TARGET_OS_TV
+  NSArray *directoryPaths =
+      NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
+#else
   NSArray *directoryPaths =
       NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
+#endif
   NSString *applicationSupportDirPath = directoryPaths.lastObject;
```

Comment:
Can you please create an issue for here and the rest of the tests to rename the method and constants away from `ApplicationSupportDirectory`? `isPlistInApplicationSupportDirectory` doesn't make sense because it's not ApplicationSupport, neither does `kApplicationSupportSubDirectoryName`. I don't think it should block merging but an issue tracking the change would be good for a follow up.